### PR TITLE
Create release.sh

### PR DESCRIPTION
**Pull request by danparisd**
This will allow us to push a git tag and that will create a release.

### CODE REVIEW

**Comment by danparisd** [comment]
This should be ready, due to it's nature there's not a good way to test it without pulling the file in and pushing a tag as if you wanted an actual release. The jenkins jobs are set up and should be ready to go. @PurpleGuitar @BryanHo10

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+if [ -z "$DOES_RELEASE_EXIST" ]; then exit;
+fi
+
+#Check for the gothub Github release go package, if not there, go get it
+if [ ! -f "$GOPATH/bin/gothub" ]; then
+go get github.com/itchio/gothub; else exit;
+fi
+
+#create the release
+gothub release --tag $TAG -p
```
Comment (BryanHo10) [inline release.sh]: @danparisd is it supossd to be gothob?

Comment (danparisd) [reply release.sh]: yeah, the go package that I'm using to push to github is called `gothub` it's confusing.

Comment (BryanHo10) [reply release.sh]: Oh nvm I see. it's the library. Looks good

**Pull request merged**